physics/PBL/SATMEDMF/satmedmfvdifq.meta

```diff
@@ -217,6 +217,29 @@
   type = real
   kind = kind_phys
   intent = in
+[usfco]
+  standard_name = ocn_current_zonal
+  long_name = ocn_current_zonal
+  units = m s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = in
+[vsfco]
+  standard_name = ocn_current_merid
+  long_name = ocn_current_merid
+  units = m s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  intent = in
+[icplocn2atm]
+  standard_name = flag_for_air_sea_flux_computation_over_water
```

Comment:
Since this is an integer control (as opposed to a logical), we're supposed to use "control_for_air_sea_flux_computation_over_water" instead. Also, the units should be 1. I realize that this hasn't been enforced with similar variables, but this is how it is supposed to be according to the metadata rules: https://github.com/ESCOMP/CCPPStandardNames/blob/main/StandardNamesRules.rst

Reply:
See comment above on logical vs. integer.